server/models/index.js

```diff
@@ -52,9 +44,9 @@ let saveWishlist = (products, username) => {
     }
   });
 };
-//get all the products
+
 let getProducts = (callback) => {
-  MyProductsModel.find({}, (err, docs) => {
+  MyProductsModel.find({}).sort([['productId', 'ascending']]).exec(function (err, docs) {
     callback(err, docs);
   });
 };
```

Comment:
I personally check if there was an error, then callback(err, null), else callback(null, docs). Never tried it this way but I assume it works since null is a falsey value when you check for the error back in server/index.js?